README.md

```diff
@@ -199,7 +199,54 @@ CDI specification, so Podman is recommended for running containers on Jetson
 devices. Docker is set to get experimental CDI support in their version 25
 release.
 
+## Configuring CUDA for Nixpkgs
+
```

Comment:
Can you explain what the default behavior of using the NixOS module is, and mention the new configureCuda option up front?

If the specifics of the cudaSupport / cudaCapabilities is documented upstream already, you can just have a link to that documentation.

Reply:
Added a note about the default behavior.

There’s also no docs on this upstream, so I guess this is the first place these guidelines have been documented.